spec/models/api/v1/query_param_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Api::V1::QueryParam do
+  let(:dummy_controller) { Api::V1::BaseController.new.extend Api::V1::QueryParam }
```

Comment:
Why extend instead of include?

Reply:
The result is the same. But updated with include syntax as agree is clearer